### PR TITLE
adding bindings for next_on_output and prev_on_output

### PR DESCRIPTION
**Pull request by ferdinandyb**
This would cover some problems mentioned in https://github.com/regolith-linux/regolith-desktop/issues/213 in a painless way.

### CODE REVIEW

**Comment by kgilmer** [comment]
Looks great, thanks @priestoferis !  One tweak that I'd like is to use either `in` or `on` rather than `within` for the comment used by Remontoire.

File: config [hunk 1]
```diff
@@ -187,6 +187,14 @@ bindsym $mod+$i3-wm.binding.ws_next workspace next
 set_from_resource $i3-wm.binding.ws_next2 i3-wm.binding.ws_next2 Mod1+Right
 bindsym $mod+$i3-wm.binding.ws_next2 workspace next
 
+## Navigate // Next Workspace within Output // <><Ctrl> Tab ##
```
Comment (kgilmer) [inline config]: Can we use `on` or '`in' here instead of `within`?

File: config [hunk 2]
```diff
@@ -187,6 +187,14 @@ bindsym $mod+$i3-wm.binding.ws_next workspace next
...
+set_from_resource $i3-wm.binding.ws_next_on_output i3-wm.binding.ws_next_on_output Ctrl+Tab
+bindsym $mod+$i3-wm.binding.ws_next_on_output workspace next_on_output
+
+## Navigate // Next Workspace within Output // <><Ctrl> l ##
```
Comment (kgilmer) [inline config]: Can we use `on` or '`in' here instead of `within`?

File: config [hunk 3]
```diff
@@ -195,6 +203,14 @@ bindsym $mod+$i3-wm.binding.ws_prev workspace prev
 set_from_resource $i3-wm.binding.ws_prev2 i3-wm.binding.ws_prev2 Mod1+Left
 bindsym $mod+$i3-wm.binding.ws_prev2 workspace prev
 
+## Navigate // Previous Workspace within Output // <><Ctrl><Shift> Tab ##
```
Comment (kgilmer) [inline config]: Can we use `on` or '`in' here instead of `within`?

File: config [hunk 4]
```diff
@@ -195,6 +203,14 @@ bindsym $mod+$i3-wm.binding.ws_prev workspace prev
...
+set_from_resource $i3-wm.binding.ws_prev_on_output i3-wm.binding.ws_prev_on_output Ctrl+Shift+Tab
+bindsym $mod+$i3-wm.binding.ws_prev_on_output workspace prev_on_output
+
+## Navigate // Previous Workspace within Output // <><Ctrl> h ##
```
Comment (kgilmer) [inline config]: Can we use `on` or '`in' here instead of `within`?

**Review by kgilmer — CHANGES_REQUESTED** [review]

File: config (hunk 1 above)
Comment (ferdinandyb) [reply config]: 'on' is I think the better choice of the two, so updated to on in all four places.

**Comment by ferdinandyb** [comment]
I updated the repo using `on` instead of `within`.

**Review by kgilmer — APPROVED** [review]
Thanks, LGTM

**Pull request merged**